keyboards/ropro/info.json

```
@@ -0,0 +1,13 @@
+{
+  "keyboard_name": "ropro",
+  "url": "https://github.com/TheRoyalSweatshirt/The_RoPro",
+  "maintainer": "[Garret G.](https://github.com/TheRoyalSweatshirt)",
```

This should just be your GitHub username.

```suggestion
  "maintainer": "TheRoyalSweatshirt",
```